devices/FloatingBaseEstimatorDevice/FloatingBaseEstimatorDevice.ini

```diff
@@ -0,0 +1,4 @@
+[plugin FloatingBaseEstimatorDevice]
+type device
+name FloatingBaseEstimatorDevice
+library FloatingBaseEstimatorDevice
```

Comment:
This file can be removed with the latest YARP feature. See https://github.com/robotology/whole-body-estimators/issues/70

Reply:
Done in https://github.com/prashanthr05/bipedal-locomotion-framework/commit/321d50d55bd80d1e46b3ab66427d2a00977ffc96

Reply:
Apparently, the YARP version is not compatible, so we will stick to the previous version for the moment.

Reply:
Should we add the following lines for the automatic ini installation to work?

```
set(YARP_FORCE_DYNAMIC_PLUGINS ON)
yarp_configure_plugins_installation(package_name)
```